.storybook/strictmode.js

```diff
@@ -0,0 +1,39 @@
+import {Checkbox} from '@react-spectrum/checkbox';
+import {defaultTheme} from './constants';
+import {makeDecorator} from '@storybook/addons';
+import {Provider} from '@react-spectrum/provider';
+import React, {StrictMode, useState} from 'react';
+
+function StrictModeDecorator(props) {
+  let {children} = props;
+  let [selected, setSelected] = useState(false);
+  let wrapper = selected ? (
+    <StrictMode>
+      {children}
+    </StrictMode>
+  ) : children;
+
+  return (
+    <Provider theme={defaultTheme}>
+      <Checkbox
+        marginTop={10}
+        isSelected={selected}
+        onChange={() => setSelected(selected => !selected)}>
+        Toggle strict mode
+      </Checkbox>
```

Comment:
TODO: still trying to figure out if it is possible to have this be a option in the `controls` panel instead

Reply:
Can't put it there, but you could put it up in the toolbar across the top of the stories, where we put the locale/scale/theme

Reply:
urgh, I was hoping that https://storybook.js.org/docs/react/essentials/controls#parameters was alluding to being able to set a control param globally but putting it in the toolbar sounds good as well

Reply:
you probably could through a decorator but I think it'd be a lot more work than it's worth, plus you'd have to apply it to every component, there's a number of them with no controls
I think toolbar is the better solution